doc/Language/classtut.pod6

```diff
@@ -112,85 +86,13 @@ my $eat =
 $eat.perform();
 =end code
 
-In this case, C<BUILD> is needed since we have overridden the default
-C<new> constructor. C<bless> is eventually invoking it with the two named
-arguments that correspond to the two properties without a default value. With
-its signature, C<BUILD> converts the two positionals to the two attributes,
-C<&!callback> and C<@!dependencies>, and returns the object (or turns it in to
-the next phase, C<TWEAK>, if available).
-
-Declaring C<new> as a C<method> and not as a C<multi method> prevents us
-from using the default constructor; this implicit constructor uses the
-attributes as named parameters. This is one of the reasons why using
-C<new> as a method's name is discouraged. If you need to declare it anyway,
-use C<multi method new> if you do not want to disable the default constructor.
-
-C<TWEAK> is the last submethod to be called, and it has the
-advantage of having the object properties available without needing to
-use the metaobject protocol. It can be used, for instance, to assign
-values to instance variables based on the values of other attributes or
-instance variables:
-
-=begin code
-class Str-with-ID is Str {
-    my $counter = 0;
-    has Str $.string;
-    has Int $.ID;
-
-    submethod TWEAK() {
-        $!ID = $counter++;
-    }
-}
-
-say Str-with-ID.new(string => 'First').ID;  # OUTPUT: «0»
-say Str-with-ID.new(string => 'Second').ID; # OUTPUT: «1»
-=end code
-
-In this case, we need to compute C<$.ID> from the value of a counter that is a
-class variable, C<$counter>, thus we simply assign a value to it and increment
-the counter at the same time. Please check also
-L<this section on C<TWEAK> in the Object Orientation (OO) document|/language/objects#index-entry-TWEAK>
-for the mechanics of object construction.
-
-X<|DESTROY>
-X<|submethod DESTROY>
-
-C<DESTROY> is the submethod that gets called when an object is garbage
-collected. That is going to happen only if the runtime needs the memory, so we
-can't rely on when it's going to happen. In particular, it could happen in the
-middle of some running code in a thread, so we must take special care to not
-assume any context during that event. We can use it to close any kind of handles
-or supplies or delete temporary files that are no longer going to be used.
-
-=begin code
-my $in_destructor = 0;
-
-class Foo {
-    submethod DESTROY { $in_destructor++ }
-}
-
-my $foo;
-for 1 .. 6000 {
-    $foo = Foo.new();
-}
-
-say "DESTROY called $in_destructor times";
-=end code
-
-This might print something like C<DESTROY called 5701 times>, but it only kicks
-in after we have stomped over former instances of C<Foo> 6000 times. We can't
-rely, however, on the order of destruction, for instance.
```

Comment:
Moved. See the corresponding comment and https://github.com/Raku/doc/commit/d9c784dc392b8e98a337e0eebfca017ed14624d4.